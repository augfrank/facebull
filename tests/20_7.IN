M0   C0   C1  2345
M1   C0   C2  2857
M2   C0   C3  2011
M3   C0   C4  2802
M4   C0   C5  2706
M5   C0   C7  2649
M6   C0   C9  2700
M7   C0   C11  2958
M8   C0   C12  2058
M9   C0   C13  2657
M10   C0   C14  2347
M11   C0   C15  2365
M12   C0   C16  2946
M13   C0   C17  2492
M14   C0   C18  2029
M15   C1   C0  2800
M16   C1   C2  2748
M17   C1   C3  2583
M18   C1   C4  2909
M19   C1   C5  2809
M20   C1   C6  2024
M21   C1   C7  2325
M22   C1   C8  2005
M23   C1   C9  2151
M24   C1   C10  2686
M25   C1   C11  2652
M26   C1   C12  2101
M27   C1   C14  2536
M28   C1   C15  2100
M29   C1   C16  2127
M30   C1   C18  2637
M31   C1   C19  2607
M32   C2   C0  2626
M33   C2   C1  2835
M34   C2   C4  2887
M35   C2   C5  2227
M36   C2   C6  2395
M37   C2   C7  2007
M38   C2   C8  2619
M39   C2   C9  2791
M40   C2   C10  2896
M41   C2   C11  2599
M42   C2   C12  2500
M43   C2   C14  2710
M44   C2   C15  2516
M45   C2   C17  2683
M46   C2   C18  2774
M47   C2   C19  2968
M48   C3   C0  2289
M49   C3   C2  2738
M50   C3   C4  2029
M51   C3   C5  2711
M52   C3   C6  2693
M53   C3   C7  2935
M54   C3   C8  2815
M55   C3   C9  2169
M56   C3   C10  2216
M57   C3   C12  2980
M58   C3   C13  2619
M59   C3   C14  2121
M60   C3   C15  2410
M61   C3   C16  2009
M62   C3   C18  2591
M63   C3   C19  2438
M64   C4   C0  2013
M65   C4   C1  2383
M66   C4   C2  2411
M67   C4   C5  2454
M68   C4   C6  2770
M69   C4   C9  2323
M70   C4   C10  2104
M71   C4   C11  2018
M72   C4   C12  2425
M73   C4   C13  2166
M74   C4   C14  2609
M75   C4   C16  2453
M76   C4   C17  2448
M77   C4   C18  2358
M78   C4   C19  2318
M79   C5   C0  2622
M80   C5   C2  2998
M81   C5   C3  2378
M82   C5   C4  2563
M83   C5   C6  2975
M84   C5   C8  2970
M85   C5   C9  2687
M86   C5   C11  2000
M87   C5   C12  2340
M88   C5   C13  2692
M89   C5   C14  2469
M90   C5   C16  2688
M91   C5   C18  2275
M92   C5   C19  2629
M93   C6   C0  2515
M94   C6   C2  2879
M95   C6   C3  2056
M96   C6   C4  2812
M97   C6   C5  2890
M98   C6   C8  2664
M99   C6   C10  2066
M100   C6   C11  2386
M101   C6   C12  2993
M102   C6   C14  2511
M103   C6   C15  2470
M104   C6   C17  2122
M105   C6   C18  2728
M106   C6   C19  2224
M107   C7   C1  2808
M108   C7   C2  2703
M109   C7   C3  2744
M110   C7   C4  2692
M111   C7   C5  2262
M112   C7   C6  2767
M113   C7   C8  2446
M114   C7   C10  2291
M115   C7   C14  2670
M116   C7   C15  2383
M117   C7   C16  2222
M118   C7   C18  2744
M119   C7   C19  2325
M120   C8   C2  2365
M121   C8   C3  2001
M122   C8   C4  2090
M123   C8   C5  2744
M124   C8   C6  2881
M125   C8   C7  2536
M126   C8   C9  2905
M127   C8   C11  2300
M128   C8   C13  2689
M129   C8   C14  2751
M130   C8   C15  2694
M131   C8   C18  2870
M132   C8   C19  2644
M133   C9   C0  2995
M134   C9   C1  2355
M135   C9   C2  2004
M136   C9   C4  2374
M137   C9   C5  2733
M138   C9   C7  2317
M139   C9   C8  2933
M140   C9   C10  2153
M141   C9   C11  2917
M142   C9   C12  2938
M143   C9   C14  2547
M144   C9   C15  2440
M145   C9   C16  2307
M146   C9   C17  2520
M147   C9   C18  2100
M148   C10   C0  2484
M149   C10   C1  2375
M150   C10   C2  2098
M151   C10   C3  2253
M152   C10   C4  2125
M153   C10   C6  2693
M154   C10   C7  2993
M155   C10   C8  2147
M156   C10   C9  2259
M157   C10   C12  2556
M158   C10   C13  2672
M159   C10   C14  2820
M160   C10   C15  2187
M161   C10   C16  2319
M162   C10   C18  2534
M163   C10   C19  2718
M164   C11   C0  2034
M165   C11   C1  2668
M166   C11   C2  2021
M167   C11   C4  2675
M168   C11   C5  2893
M169   C11   C6  2348
M170   C11   C7  2368
M171   C11   C9  2830
M172   C11   C10  2114
M173   C11   C12  2206
M174   C11   C13  2828
M175   C11   C14  2591
M176   C11   C15  2215
M177   C11   C16  2965
M178   C11   C18  2381
M179   C11   C19  2975
M180   C12   C0  2979
M181   C12   C1  2217
M182   C12   C2  2446
M183   C12   C3  2089
M184   C12   C4  2920
M185   C12   C6  2478
M186   C12   C8  2323
M187   C12   C9  2379
M188   C12   C10  2164
M189   C12   C11  2382
M190   C12   C13  2647
M191   C12   C15  2094
M192   C12   C16  2051
M193   C12   C17  2999
M194   C12   C18  2729
M195   C13   C1  2860
M196   C13   C2  2308
M197   C13   C3  2873
M198   C13   C4  2314
M199   C13   C5  2945
M200   C13   C7  2057
M201   C13   C8  2158
M202   C13   C9  2834
M203   C13   C10  2141
M204   C13   C11  2860
M205   C13   C12  2707
M206   C13   C14  2135
M207   C13   C15  2185
M208   C13   C16  2273
M209   C13   C17  2470
M210   C13   C18  2047
M211   C13   C19  2483
M212   C14   C0  2262
M213   C14   C1  2819
M214   C14   C2  2442
M215   C14   C3  2297
M216   C14   C4  2385
M217   C14   C6  2453
M218   C14   C8  2436
M219   C14   C10  2270
M220   C14   C11  2505
M221   C14   C12  2269
M222   C14   C13  2298
M223   C14   C15  2237
M224   C14   C17  2389
M225   C15   C0  2059
M226   C15   C2  2971
M227   C15   C3  2341
M228   C15   C6  2828
M229   C15   C7  2491
M230   C15   C8  2596
M231   C15   C9  2853
M232   C15   C11  2958
M233   C15   C12  2476
M234   C15   C13  2478
M235   C15   C14  2185
M236   C15   C16  2426
M237   C15   C17  2995
M238   C15   C18  2355
M239   C15   C19  2932
M240   C16   C0  2875
M241   C16   C1  2908
M242   C16   C2  2046
M243   C16   C3  2108
M244   C16   C5  2245
M245   C16   C6  2773
M246   C16   C7  2532
M247   C16   C10  2467
M248   C16   C11  2521
M249   C16   C12  2041
M250   C16   C13  2004
M251   C16   C14  2952
M252   C16   C15  2902
M253   C16   C18  2527
M254   C17   C0  2378
M255   C17   C1  2804
M256   C17   C2  2314
M257   C17   C3  2218
M258   C17   C4  2619
M259   C17   C5  2731
M260   C17   C6  2512
M261   C17   C7  2821
M262   C17   C8  2705
M263   C17   C10  2618
M264   C17   C11  2455
M265   C17   C12  2724
M266   C17   C13  2614
M267   C17   C14  2356
M268   C17   C15  2040
M269   C17   C16  2662
M270   C17   C18  2758
M271   C17   C19  2291
M272   C18   C0  2866
M273   C18   C2  2402
M274   C18   C3  2639
M275   C18   C5  2377
M276   C18   C6  2347
M277   C18   C7  2026
M278   C18   C9  2452
M279   C18   C12  2469
M280   C18   C13  2508
M281   C18   C14  2432
M282   C18   C15  2207
M283   C18   C16  2152
M284   C18   C19  2783
M285   C19   C0  2276
M286   C19   C2  2388
M287   C19   C3  2284
M288   C19   C4  2867
M289   C19   C5  2911
M290   C19   C6  2708
M291   C19   C7  2337
M292   C19   C8  2232
M293   C19   C9  2316
M294   C19   C12  2434
M295   C19   C14  2979
M296   C19   C15  2497
M297   C19   C16  2949
M298   C19   C17  2066
M299   C19   C18  2458
