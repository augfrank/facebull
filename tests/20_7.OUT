42114
2 20 35 61 69 86 99 119 122 135 155 164 181 210 221 235 250 268 277 298
